tests/tx/test_indexes3.py

```diff
@@ -62,24 +60,20 @@ def test_tips_index_initialization(self):
         base_block_tips_tree = tx_storage.indexes.block_tips.tree.copy()
         base_tx_tips_tree = tx_storage.indexes.tx_tips.tree.copy()
 
-        # reset the indexes and force a manual initialization
-        tx_storage._reset_cache()
-        self.manager._initialize_components()
+        # reset the indexes, which will force a re-initialization of all indexes
+        tx_storage._manually_initialize()
 
         reinit_all_tips_tree = tx_storage.indexes.all_tips.tree.copy()
```

Comment:
Instead of doing each one manually, should we iterate over all indexes?

Reply:
I don't think that works very well. This is implementation specific, and iterating over indexes means erasing the specific class and only looking at the interface.

I don't think it'd be good to make all indexes implement some form of copying or serialization that can be used for comparisons.

Maybe there could be some improvement we could make in this front, but I think it would need better thought, I'd rather avoid this changing this right now.

Reply:
Makes sense. Thanks!